src/main/scala/cc/factorie/app/nlp/ner/StackedChainNer.scala

```diff
@@ -585,55 +585,10 @@ class StackedChainNer[L<:NerTag](labelDomain: CategoricalDomain[String],
   }
 }
 
-object StackedChainNer {
-  def initLexicons() : Unit = synchronized {
-    lexicon.iesl.Month.toString()
-    lexicon.iesl.Day.toString()
-
-    lexicon.iesl.PersonFirst.toString()
-    lexicon.iesl.PersonFirstHigh.toString()
-    lexicon.iesl.PersonFirstHighest.toString()
-    lexicon.iesl.PersonFirstMedium.toString()
-
-    lexicon.iesl.PersonLast.toString()
-    lexicon.iesl.PersonLastHigh.toString()
-    lexicon.iesl.PersonLastHighest.toString()
-    lexicon.iesl.PersonLastMedium.toString()
-
-    lexicon.iesl.PersonHonorific.toString()
-
-    lexicon.iesl.Company.toString()
-    lexicon.iesl.JobTitle.toString()
-    lexicon.iesl.OrgSuffix.toString()
-
-    lexicon.iesl.Country.toString()
-    lexicon.iesl.City.toString()
-    lexicon.iesl.PlaceSuffix.toString()
-    lexicon.iesl.USState.toString()
-    lexicon.iesl.Continents.toString()
-
-    lexicon.wikipedia.Person.toString()
-    lexicon.wikipedia.Event.toString()
-    lexicon.wikipedia.Location.toString()
-    lexicon.wikipedia.Organization.toString()
-    lexicon.wikipedia.ManMadeThing.toString()
-    lexicon.iesl.Demonym.toString()
-
-    lexicon.wikipedia.Book.toString()
-    lexicon.wikipedia.Business.toString()
-    lexicon.wikipedia.Film.toString()
-
-    lexicon.wikipedia.LocationAndRedirect.toString()
-    lexicon.wikipedia.PersonAndRedirect.toString()
-    lexicon.wikipedia.OrganizationAndRedirect.toString()
-  }
-}
-
```

Comment:
Just jumping in here to say a couple of things. Fundamentally, I'm not sure why the loading of the lexicons is tied into any particular model. For example, if I have two NER models (one for news text, one for Twitter, for example) and I have two threads loading those models, how is that going to affect the loading of the lexicons? Will it be thread safe? Will I end up with two copies of each lexicon?

I guess my problem boils down to the fact that the failure mode of trying to automagically load a lexicon when loading a model that uses that lexicon is "What the hell just happened?" It's hard (impossible?) to anticipate all of the environments where models will be loaded, and it appears that some cases will just break with no recourse for the developer to fix it.

I would strongly prefer a model where it is up to the developer to load the lexicons that are needed for whatever problem she is working on. These large, global resources can then be loaded safely (because the developer is aware of the environment where the loading will occur.) If the developer forgets to load a particular lexicon and then tries using a model that relies on that lexicon, then that's a bug that needs to be fixed.